starter-code/index.html

```diff
@@ -0,0 +1,226 @@
+                    <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra</p>
+                    <button type="button" class="btn btn-outline-secondary">READ MORE</button>
+                </div>
+            </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+        </a>
+    </div>
+
+    <div class="red">
+        <div class="row">
+            <div class="col-sm-4">
+                <p><img class="rot" src="img/A.png" alt="A">
+                    <h3>Web design</h3>
+                    <p>
+                        Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius.
+                    </p>
+            </div>
+            <div class="col-sm-4">
+                <p><img class="rot" src="img/B.png" alt="B">
+                    <h3>Web development</h3>
+                    <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat luctus. </p>
+            </div>
+            <div class="col-sm-4">
+                <p><img class="rot" src="img/C.png" alt="C">
+                    <h3>User interface</h3>
+                    <p>Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien dignissim, eu dapibus mi molestie. Suspendisse dictum convallis quam et sodales. Praesent non enim et magna congue gravida. </p>
+            </div>
+        </div>
+    </div>
+
+    <div class="container projects">
+        <div class="row">
+            <div class="title">
+                <h2>Latest Projects</h2>
+                <p>Few of the latest portfolio items </p>
+            </div>
+
+            <div class="pics">
+                <div class="col-sm-3 col-6">
+                    <a href="#"><img src="img/project-1.jpg" alt="1"> Nunc libero elit, iaculis nec augue non
+                    </a>
+                </div>
+                <div class="col-sm-3 col-6"><img src="img/project-2.jpg" alt="2">
```

Comment:
Notice that you are sometimes nesting <img> in <a>, and sometimes the other way around. This could be the reason for the layout problems with your lower images and their responsiveness.